Resources/Prototypes/_Harmony/Body/Prototypes/vulpkanin.yml

```
@@ -0,0 +1,49 @@
+﻿- type: body
+  id: Vulpkanin
+  name: "vulpkanin"
+  root: torso
+  slots:
+    head:
+      part: HeadVulpkanin
+      connections:
+      - torso
+      organs:
+        brain: OrganHumanBrain
+        eyes: OrganHumanEyes
+    torso:
+      part: TorsoVulpkanin
+      connections:
+      - left arm
+      - right arm
+      - left leg
+      - right leg
+      organs:
+        heart: OrganHumanHeart
```

```suggestion
        heart: OrganAnimalHeart
```
Needs an animal heart instead of the human one

---

human heart has been swapped with the appropriate animal heart